backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/Context.kt

```diff
@@ -436,6 +436,8 @@ internal class Context(config: KonanConfig) : KonanBackendContext(config) {
     fun shouldProfilePhases() = config.phaseConfig.needProfiling
 
     fun shouldContainDebugInfo() = config.debug
+    fun shouldContainLocationDebugInfo() = shouldContainDebugInfo() || config.lightDebug
+    fun shouldContainSomeDebugInfo() = shouldContainDebugInfo() || shouldContainLocationDebugInfo()
```

Comment:
Not very clear, maybe shouldContainSomeDebugInfo -> shouldContainBasicDebugInfo?

Reply:
The logic behind this name is: "if some debug info is emitted, then we must generate a DI header and properly finalize it".

Reply:
Then maybe `shouldContainAnyDebugInfo`?